public/locales/de/bridge.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
de?
I thought we were only supporting en!!

Reply:
We have other languages that are semi-translated. Untranslated keys outside EN must be in english.